packages/dev/core/src/Materials/meshDebugPluginMaterial.ts

```diff
@@ -103,6 +148,52 @@ flat varying float dbg_vPass;
 #endif
 #endif`;
 
+const fragmentDefinitionsWebGPU = `#if defined(DBG_ENABLED)
+varying dbg_vBarycentric: vec3f;
+varying dbg_vVertexWorldPos: vec3f;
+varying dbg_vPass: f32;
+
+#if !defined(DBG_MULTIPLY)
+    fn dbg_applyShading(color: vec3f) -> vec3f {
+        var N = input.vNormalW.xyz;
+        var L = normalize(scene.vEyePosition.xyz - input.vPositionW.xyz);

```

Comment:
variable `input` is out of scope in this context
use fragmentInputs instead

Reply:
nice ones! thanks I will fix that !